internal/cmd/base/servers.go

```diff
@@ -122,9 +122,9 @@ type Server struct {
 	DevSecondaryTargetId             string // Target using host sources.
 	DevHostAddress                   string // Host address for target using host sources.
 	DevTargetAddress                 string // Network address for target with address.
-	DevTargetDefaultPort             int
-	DevTargetSessionMaxSeconds       int
-	DevTargetSessionConnectionLimit  int
+	DevTargetDefaultPort             int64
```

Comment:
uint16?

Reply:
The only blocker to changing this to `uint16` is that we then couldn't pass a pointer to this variable to all the `f.Uint64Var` calls we do.

Reply:
All done - See [df25e75](https://github.com/hashicorp/boundary/pull/5027/commits/df25e75990daedd9b3b6620f6bd9eb8e8855d011) and https://github.com/hashicorp/boundary/pull/5027/commits/b0880a8fd941ca3db45cad5273d15c88bc0730eb